dts/x86/qemu_x86.dts

```diff
@@ -0,0 +1,33 @@
+/dts-v1/;
+
+#include <ia32.dtsi>
+
+/ {
+	model = "QEMU X86";
+	compatible = "intel,ia32-qemu", "intel,ia32";
```

Comment:
not sure we should claim this as "intel,ia32-qemu", but not sure what it should be.